.github/config.yml

```diff
@@ -0,0 +1,42 @@
+# Configuration for the-welcome-bot - https://github.com/bash-bot/the-welcome-bot
```

Comment:
this change  should not be committed yaml is configuration not documentation file...